pop-api/examples/fungibles/tests.rs

```diff
@@ -818,13 +843,13 @@ mod utils {
 		PSP22Error::Custom(status_code.to_string())
 	}
 
-	/// This is used to resolve type mismatches between the `AccountId` in the quasi testing
-	/// environment and the contract environment.
+	// This is used to resolve type mismatches between the `AccountId` in the quasi testing
```

Comment:
```suggestion
	/// This is used to resolve type mismatches between the `AccountId` in the quasi testing
```
Shouldn't this is `///`?